opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -620,3 +620,31 @@ excludeServiceMonitorsFromRemotePoller=DHCP,NSClient,RadiusAuth,XMP
 #   placeholder
 #   backshift
 #org.opennms.web.graphs.engine=png
+
+# ###### GRAFANA BOX ######
+# This setting controls whether a grafana box showing the available dashboards is
+# placed on the landing page. The two valid options for this are 'true' or 'false'
+# which is the default value.
+#org.opennms.grafanaBox.show=false
+#
+# If the box is enabled you also need to specify hostname of the grafana server
+#org.opennms.grafanaBox.hostname=localhost
+#
+# And also the port of the grafana server
+#org.opennms.grafanaBox.port=3000
+#
+# The API key is needed for the ReST calls to work
+#org.opennms.grafanaBox.apiKey=
+#
+# When a tag is specified only dashboards with this given tag
+# will be displayed. When no tag is given all dashboards will
+# be displayed.
+#org.opennms.grafanaBox.tag=
+#
+# The protocol for the ReST call can also be specified
+#org.opennms.grafanaBox.protocol=http
+#
+# Timeouts for contacting the grafana server
+#org.opennms.grafanaBox.connectionTimeout=500
+#org.opennms.grafanaBox.soTimeout=500
```

Comment:
It's the timeout on the socket level. JavaDoc for this:

Enable/disable SO_TIMEOUT with the specified timeout, in milliseconds. With this option set to a non-zero timeout, a read() call on the InputStream associated with this Socket will block for only this amount of time. If the timeout expires, a java.net.SocketTimeoutException is raised, though the Socket is still valid. The option must be enabled prior to entering the blocking operation to have effect. The timeout must be > 0. A timeout of zero is interpreted as an infinite timeout.